src/js/components/path/index.tsx

```diff
@@ -7,16 +7,18 @@ interface PathProps {
   activeIdx?: number;
   isCompleted?: boolean;
   isLost?: boolean;
+  label?: string;
 }
 
-const Path = ({ steps, activeIdx, isCompleted, isLost }: PathProps) => (
+const Path = ({ steps, activeIdx, isCompleted, isLost, label }: PathProps) => (
   <div className="slds-region_small">
     <div className="slds-path">
       <div className="slds-grid slds-path__track">
         <ul
           className="slds-path__nav"
           role="listbox"
           aria-orientation="horizontal"
+          aria-label={label}
```

Comment:
Is an empty string a valid value for `aria-label`? Just noting that `label` is nullable - do we need to worry about that?

Reply:
Empty `aria-labels` are ignored, but great question!